ggml.c

```diff
@@ -1123,12 +1124,94 @@ static void quantize_row_q4_2_reference(const float * restrict x, block_q4_2 * r
     }
 }
 
+static inline int nearest_int(float fval) {
```

Comment:
this `inline` does not do anything here. the `static` is all you need.

Reply:
hm actually, after looking at cppref, i am not sure that C and C++ are the same here.